jobs/rootfs-builder.jpl

```diff
@@ -46,6 +46,9 @@ import org.kernelci.util.Job
 
 def build(config, arch, pipeline_version, kci_core, rootfs_type) {
     dir(kci_core) {
+        sh 'id'
+        sh 'sudo id'
+        sh 'df -h'
```

Comment:
This looks like some debug commands, can they be dropped?

Reply:
I left them intentionally, if job fails, to know for sure if it is because id is changed, or it might be disk space issue, or sudo doesnt work anymore. As running job again will be quite expensive in matters of time and computing resources. Sure i can remove, if its bad idea. Or i can add comment about this reason.

Reply:
I see, yes I think having a comment would be good to explain why this is particularly useful information to print.